docs/Product-Guides/Benefits-API.md

```diff
@@ -1,85 +1,107 @@
 
-For Automated connections that have Assisted Benefits enabled, the response you receive from the `/employer/benefits` endpoints will contain a `job_id`. While our dedicated product operations team makes the necessary changes in the provider's system, you can continue to call the `GET /jobs/{job_id}` to get the status of the job. The valid job status responses will be either `pending`, `in_progress`, `complete`, or `error` with a response body further explaining the response.
+### Updating existing enrollments
 
-Example `POST /employer/benefits` response:
+To update enrollment configurations for currently enrolled individuals, you can use the same `POST /benefits/{benefit_id}/individuals` endpoint that you use to enroll new individuals. When updating an existing enrollment for an individual, the enrollment configuration will be completely overwritten with the new configuration provided in the request, so please make sure to include the entire desired configuration. Since they both use the same endpoint, these update requests can be submitted in the same batch request as new enrollment requests.
+
+### Un-enrolling individuals
+
+You can unenroll individuals by using the [`DELETE /benefits/{benefit_id}/individuals`](https://developer.tryfinch.com/docs/reference/b3A6MjE4NDU0NTY-unenroll-individuals-from-benefits) endpoint. This will remove the enrollment configuration for an individual.
+
+## Automated vs Assisted Connections
+
+Requests made for Benefits connections to automated providers will be fulfilled synchronously, so the response you receive from the `/employer/benefits` endpoints will contain the results of the request. For providers which support Benefits via Assisted, the response you receive from the `/employer/benefits` endpoints will contain a `job_id`. While our dedicated product operations team makes the necessary changes in the provider's system, you can continue to call  `GET /jobs/{job_id}` to get the status of the job. The valid job status responses will be either `pending`, `in_progress`, `complete`, or `error` with a response body further explaining the response.
+
+Example `POST /employer/benefits` response:
 
 ```json
 {
   "benefit_id": "264122c0-0216-4f21-a4cd-7b3bcddbe3fc",
   "job_id": "497d98f3-580a-4ab9-830a-af2346d029b2"
 }
+
 ```
 
-Example `POST /employer/benefits/{benefit_id}/individuals` Response:
+Example `POST /employer/benefits/{benefit_id}/individuals` Response:
```

Comment:
We might want to change `Response:` to lower-case here to match what we have on line 72 (or vice versa)